routes/tasks.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express')
+const router = express.Router()
+const tasksController = require('../controllers/tasksController')
+
+/**
+ * @swagger
+ * /tasks:
+ *  get:
+ *   summary: Used to get all the tasks
+ *   responses:
+ *     200:
+ *       description: returns tasks
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/tasks'
+ *     404:
+ *       description : no tasks found
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/notFound'
+ *     500:
+ *       description: badImplementation
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/badImplementation'
+ */
+
+router.get('/', tasksController.fetchTasks)
+
+/**
+ * @swagger
+ * /tasks:
+ *  post:
+ *   summary: Used to create new task
+ *   responses:
+ *     200:
+ *       description: returns newly created task
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/tasks'
+ *     500:
+ *       description: badImplementation
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.post('/', tasksController.addNewTask)
+
+/**
+ * @swagger
+ * /tasks:
+ *  patch:
+ *   summary: Used to update task details
```

Comment:
- For the patch route we are sending the data which needs to be updated right ? So, we need to mention it here in swagger docs. Please add `requestBody` after summary. 
- You can refer this: https://swagger.io/docs/specification/basic-structure/

Rest everything looks good to me! Thank you. 😃

Reply:
Done, Please check.